juriscraper/opinions/united_states/federal_bankruptcy/bap1.py

```diff
@@ -0,0 +1,134 @@
+"""
+Scraper for the United States Bankruptcy Appellate Panel for the First Circuit
+CourtID: bap1
+Court Short Name: 1st Cir. BAP
+"""
+import calendar
```

Comment:
Lets update the notes at the top of the page to match this pattern 

```
"""
Scraper for Florida 5th District Court of Appeal
CourtID: flaapp5
Court Short Name: flaapp5
Court Contact: 5dca@flcourts.org, 386-947-1530
Author: Andrei Chelaru
History:
 - 2014-07-23, Andrei Chelaru: Created.
 - 2014-08-05, mlr: Updated.
 - 2014-08-06, mlr: Updated.
 - 2014-09-18, mlr: Updated date parsing code to handle Sept.
 - 2016-03-16, arderyp: Updated to return proper absolute pdf url paths, simplify date logic
"""
```

there are variations around - but I think its nice to have history - and author included